src/app/index.ts

```diff
@@ -62,16 +62,7 @@ app.use(
     // might consider turning this off to allow individual front-ends set Content-Security-Policy on meta tags themselves if list grows long
     // <meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline'  https://cdnjs.cloudflare.com;" >
     contentSecurityPolicy: {
-      directives: {
```

Comment:
@machariamuguku I  think there is some contextual coupling between the different `-src` directives. i.e. supplying a certain directive like `script-src`: would mean you are overriding the `default-src`. The I way I see it, is that if you are in a position to define both then it's going to be much more straightforward to think about how all your directive values should look like.

I also think the `default-src` is one of the directives that is most likely to change.

Reply:
@p-netm I agree. All I'm saying is that `default-src` or whichever directives are being defined/overridden should be in the code base and then only provide the override values as env's. So instead of providing this `{"default-src":["'self'"]}` for example, we'll only provide `["self"]` which will be spread into the directive

Reply:
@muguku what would your proposed change look like?

Reply:
@p-netm something like this:

'default-src': [
"'self'",
...ENV_STR_WITH_LINKS_SEPERATED_WITH_COMA.split(",")
],

Reply:
@machariamuguku would we then require atleast 2 envs one for `default-src`, and another for other `-src`?